app/pods/components/web-components/au-textarea/template.hbs

```diff
@@ -1,8 +1,8 @@
-<Textarea 
-  class="auk-textarea auk-textarea--block"
+<Textarea
+  class="auk-textarea auk-textarea--block {{resize}}"
```

Comment:
Hier wil je this.resize
Nu is het onduidelijk omdat zowel @resize (de inkomende value)  als **this.resize** bestaan (de css klasse) bestaan in de template
Blijkbaar neemt ember hier achter de schermen **this.resize** dus komt het toch goed

Reply:
fixed